ui/src/app/components/webcomponents/dry-run-form.component.ts

```diff
@@ -225,25 +224,26 @@ export class DryRunFormComponent extends BaseComponent implements OnInit {
   }
 
   setConfigurationId(configuration: AdhocRunConfiguration) {
-    this.refresh= false;
+    this.refresh = false;
     this.configuration = undefined;
-    if(configuration?.id) {
+    if (configuration?.id) {
       this.configuration = configuration;
-      if(configuration!=null && configuration.platformOsVersionId!=null) {
+      if (configuration != null && configuration.platformOsVersionId != null) {
```

Comment:
this block get executed, if `configuration?.id`, right? if so, why do we need null check here? @manikanta-reddy-t

Reply:
Done